### DISCUSSION TITLE
Query on feature - "Apply mitigation Association Rules"

### DISCUSSION THREAD

**Discussion by rsrinivasanhome** [simonec73/threatsmanager#36] [General]
What is use of the - "Apply mitigation Association Rules" feature ?  I could not the find info in the help

![image](https://user-images.githubusercontent.com/31912418/195994320-e563b7b9-b0f0-4638-aefc-daaf2dbbf6fb.png)

Is it possible to use a Rule to assign a mitigation ?

**Comment by simonec73** [comment]
It applies the rules to generate the mitigations for the selected Threat Event.
